src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -166,21 +188,31 @@ class SidebarLinks extends React.Component {
                             styles.alignItemsCenter,
                         ]}
                         >
-                            <Header textSize="large" title="Chats" />
+                            <Header textSize="large" title={this.props.title} />
                             <TouchableOpacity
                                 style={[styles.flexRow, styles.ph5]}
                                 onPress={() => ChatSwitcher.show()}
                             >
                                 <Icon src={MagnifyingGlass} />
                             </TouchableOpacity>
-                            <TouchableOpacity
-                                onPress={this.props.onAvatarClick}
-                            >
-                                <AvatarWithIndicator
-                                    source={this.props.myPersonalDetails.avatarURL}
-                                    isActive={this.props.network && !this.props.network.isOffline}
-                                />
-                            </TouchableOpacity>
+                            {this.props.showCloseButton && (
```

Comment:
If we need to add things like `showCloseButton` and `showAvatar` then it might make more sense to reduce the responsibility of these components and use a version that allows adding in the header parts where they are needed instead of a new prop to make optional.